app/controllers/books_controller.rb

```diff
@@ -25,7 +25,7 @@ def create
 
     respond_to do |format|
       if @book.save
-        format.html { redirect_to book_url(@book), notice: 'Book was successfully created.' }
+        format.html { redirect_to book_url(@book), notice: t('books.create.notice.create') }
```

Comment:
```suggestion
        format.html { redirect_to book_url(@book), notice: t('books.create.notice') }
```

createが2回登場すると「馬から落ちて落馬して」感がありますね

Reply:
確かにそうですね...🐴
次のように修正いたしました。

```
message:
  books:
    created: "正常に作成されました"
    updated: "正常に更新されました"
    destroyed: "正常に削除されました"
```

下のように修正することも考えましたが、messageが何度も出てくることに冗長さを感じたため、上を採用しました。
直感的な読みやすさはこちらだと思いますが、迷いました。
```
books:
  create:
    message: "正常に作成されました"
  update:
    message: "正常に更新されました"
  destroy:
    message: "正常に削除されました"
```